shortfin/tests/apps/llm/components/invocation_test.py

```diff
@@ -178,7 +179,7 @@ def result_logits_none_indices_decode(decode_task, fiber):
     """Fixture to create a result logits device array."""
     vocab_size = 16
     batch_size = decode_task.req_count
-    seq_len = max(len(req.input_token_ids) for req in decode_task._exec_requests)
+    seq_len = decode_task._task_input.batch_seq_len
```

Comment:
Its usually a pretty  bad sign to have a test reaching into classes to grab implementation specific members.  Is there a better way to do this?